lib/class-wp-rest-gutenberg-blocks.php

```diff
@@ -0,0 +1,374 @@
+<?php
+/**
+ * Rest API endpoints for Gutenberg blocks.
+ */
+class WP_Gutenberg_Block_Controller extends WP_REST_Controller {
```

Comment:
The filename should match the class name, so this should be `class-wp-gutenberg-block-controller.php` instead.

Reply:
will change